wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -8,38 +8,79 @@
 {% block content %}
     <div id="comments"></div>
     {% page_permissions page as page_perms %}
-    <header class="merged tab-merged">
-        {% explorer_breadcrumb page page_perms=page_perms show_header_buttons=True %}
 
-        <div class="row">
-            <h1 class="left col9 header-title" title="{% blocktrans with title=page.get_admin_display_title page_type=content_type.model_class.get_verbose_name %}Editing {{ page_type }}{% endblocktrans %}">
-                {{ page.get_admin_display_title }}
-            </h1>
+    {# Note: padding left on mobile to give space for navigation toggle, z index 99 to ensure header is always above  #}
+    <header class="w-flex w-flex-col sm:w-flex-row w-items-center w-justify-between w-bg-gray-50 w-py-0 w-mb-0 w-relative w-top-0 w-z-[99] sm:w-sticky w-min-h-[50px]">
```

Comment:
Can you make a template for this in `wagtailadmin/shared/editor_header.html`? Or some better name?